core/src/main/kotlin/org/evomaster/core/search/StructuralElement.kt

```diff
@@ -169,6 +176,7 @@ abstract class StructuralElement (
      */
     open fun addChild(child: StructuralElement){
         verifyChildrenToInsert(child)
+        preChildrenSetup(listOf(child))
```

Comment:
shouldn't this be done after checking validity of inputs?

Reply:
This was changed by @ZhangMan1126